adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -357,10 +376,17 @@ public void acquireToken(Activity activity, String resource, String clientId,
         if (checkPreRequirements(resource, clientId, callback)) {
             redirectUri = getRedirectUri(redirectUri);
 
+            final String requestId = Telemetry.registerNewRequest();
+            final APIEvent apiEvent = createApiEvent(mContext, clientId, requestId, EventStrings.ACQUIRE_TOKEN_4);
+            apiEvent.setSilentRequestPromptBehavior(prompt.toString());
```

Comment:
misnamed function. :(

---
In reply to: [90354517](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90354517) [](ancestors = 90354517)